depends/hosts/darwin.mk

```diff
@@ -1,9 +1,8 @@
 OSX_MIN_VERSION=10.12
-OSX_SDK_VERSION=10.11
```

Comment:
16392 needs changes in macdeploy/README.md

Reply:
Resolved